spec/helpers/application_helper_spec.rb

```diff
@@ -61,7 +61,7 @@ def current_user
       end
     end
 
-    it 'inclues jquery_ujs.js' do
+    it 'includes jquery_ujs.js' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.